packages/core/handle/handle.js

```diff
@@ -92,10 +92,20 @@ import {is} from '../keymap';
  * @callback HandlerFunction
  * @memberof core/handle
  * @param {Object<string, any>} event
- * @param {any} props
+ * @param {Object<string, any>} props
  * @param {Object<string, any>} context
```

Comment:
@ryanjduffy Should we update the typedef for this to set the return type?  Seems to be working fine without it, but it seems a touch odd to me.

Reply:
If we wanted to be strict, we could document the return as `Boolean | undefined`.